05.wc/wc.rb

```diff
@@ -0,0 +1,70 @@
+# ! /usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'optparse'
+
+DDISPLAY_ADJUST_LENGTH = 1
+MULTIPLE_DISPLAY = 1
+TOTAL = '合計'
+
+def main
+  options = parse_options(ARGV)
+  output_texts = []
+  ARGF.each(nil) do |input_text|
+    wc_info = {}
+    file_name = ARGF.filename
+    wc_info[:filename] = file_name
+    wc_info[:line] = input_text.lines.count if options[:l]
+    wc_info[:word] = input_text.split(/\s+/).size if options[:w]
+    wc_info[:bytesize] = input_text.size if options[:c]
+    output_texts << wc_info
+  end
+  output_texts << file_total_info(output_texts) if output_texts.size > MULTIPLE_DISPLAY
```

Comment:
2以上なら複数（multiple）というのはある意味当たり前なので、MULTIPLE_DISPLAYという定数名はこれといった情報を提供していないことになります。
なので、定数をなくしても構わないかも？
```suggestion
  output_texts << file_total_info(output_texts) if output_texts.size > 1
```